cuebot/src/test/java/com/imageworks/spcue/test/util/CoreSpanTests.java

```diff
@@ -133,10 +135,13 @@ public void testBuildVirtualProc() {
         host.idleMemory = CueUtil.GB8;
         host.cores = 800;
         host.idleCores = 800;
+        // Hardcoded value of dispatcher.memory.mem_reserved_default
+        // to avoid having to read opencue.properties on a test setting
```

Comment:
Same recommendation: Use cuebot/src/test/resources/opencue.properties for the value, rather than hardcoding it.